internal/utils/terraform/dashboard.go

```diff
@@ -0,0 +1,141 @@
+package terraform
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+)
+
+var (
+	widgetTypes = map[string]string{
+		"viz.area":      "widget_area",
+		"viz.bar":       "widget_bar",
+		"viz.billboard": "widget_billboard",
+		"viz.bullet":    "widget_bullet",
+		"viz.funnel":    "widget_funnel",
+		"viz.heatmap":   "widget_heatmap",
+		"viz.histogram": "widget_histogram",
+		"viz.json":      "widget_json",
+		"viz.line":      "widget_line",
+		"viz.markdown":  "widget_markdown",
+		"viz.pie":       "widget_pie",
+		"viz.table":     "widget_table",
+	}
+)
+
+type Dashboard struct {
```

Comment:
Is the data structure different when exporting the JSON? Just curious about the relationship with the types that exist in the client.